packages/gatsby/src/schema/infer-graphql-input-fields.js

```diff
@@ -210,6 +210,8 @@ function inferGraphQLInputFields({
 const EXCLUDE_KEYS = {
   parent: 1,
   children: 1,
+  $loki: 1,
+  meta: 1,
```

Comment:
I realize that `meta` is loki generated field, but adds some limitation we previously didn't have - is it possible to swap to different fieldname? something more "obscure" like `$meta` that is less likely to be used by nodes?

Reply:
Great catch. I'll see what the options are in loki